synapse/lib/spawn.py

```diff
@@ -473,12 +474,14 @@ async def _delStormSvcPkgs(self, iden):
         '''
         pass
 
-    async def _runStormSvcAdd(self, iden):
+    async def _hndladdStormPkg(self, pdef):
```

Comment:
`_onAddStormPkg` ?

Reply:
nope - this has to match what is going to be called by the StormSvcClient 1:1 so we don't push changes through the nexus